src/Network/Network.Test/ScenarioTests/NetworkManagerTests.ps1

```diff
@@ -698,4 +698,275 @@ function Test-NetworkManagerResourceMinimumParameterCreate
+
+        $ruleCollection = Get-AzNetworkManagerRoutingRuleCollection -ResourceGroupName $rgname -NetworkManagerName $networkManagerName -ConfigName $RoutingConfigurationName -Name $RuleCollectionName
+        Assert-NotNull $ruleCollection;
+        Assert-AreEqual $RuleCollectionName $ruleCollection.Name;
+        Assert-AreEqual $networkGroup.Id $ruleCollection.AppliesTo[0].NetworkGroupId;
+        Assert-AreEqual "False" $ruleCollection.DisableBgpRoutePropagation;
+
+        $ruleCollection.Description = "Sample rule Collection Description"
+        $ruleCollection = Set-AzNetworkManagerRoutingRuleCollection -InputObject $ruleCollection
+        Assert-NotNull $ruleCollection;
+        Assert-AreEqual $RuleCollectionName $ruleCollection.Name;
+        Assert-AreEqual "Sample rule Collection Description" $ruleCollection.Description;
+ 
+        $destination = New-AzNetworkManagerRoutingRuleDestination -DestinationAddress "10.1.1.1/32" -Type "AddressPrefix" 
+        $nextHop = New-AzNetworkManagerRoutingRuleNextHop -NextHopType "VirtualAppliance" -NextHopAddress "2.2.2.2"
+
+        New-AzNetworkManagerRoutingRule -ResourceGroupName $rgname -NetworkManagerName $networkManagerName -ConfigName $RoutingConfigurationName -RuleCollectionName $RuleCollectionName -ResourceName $RuleName -Destination $destination -NextHop $nextHop
+
+        $routingRule = Get-AzNetworkManagerRoutingRule -ResourceGroupName $rgname -NetworkManagerName $networkManagerName -ConfigName $RoutingConfigurationName -RuleCollectionName $RuleCollectionName -Name $RuleName
+        Assert-NotNull $routingRule
+        Assert-AreEqual $RuleName $routingRule.Name 
+
+        $newRoutingRule = Set-AzNetworkManagerRoutingRule -InputObject $routingRule
+        Assert-NotNull $newRoutingRule;
+        Assert-AreEqual $RuleName $newRoutingRule.Name;
+
+        $configIds  = @($routingConfig.Id)
+        $regions = @($rglocation)  
+        Deploy-AzNetworkManagerCommit -ResourceGroupName $rgname -Name $networkManagerName -TargetLocation $regions -ConfigurationId $configIds -CommitType "Routing" 
+
+        Start-TestSleep -Seconds 60
+       
+        $deploymentStatus = Get-AzNetworkManagerDeploymentStatus -ResourceGroupName $rgname -NetworkManagerName $networkManagerName -Region $regions -DeploymentType "Routing"
+        Assert-NotNull $deploymentStatus;
+        Assert-AreEqual "Routing"  $deploymentStatus.Value[0].DeploymentType;
+        Assert-AreEqual $routingConfig.Id  $deploymentStatus.Value[0].ConfigurationIds[0];
+
+        Deploy-AzNetworkManagerCommit -ResourceGroupName $rgname -Name $networkManagerName -TargetLocation $regions -CommitType "Routing" 
+
+        Start-TestSleep -Seconds 60
+
+        $job = Remove-AzNetworkManagerRoutingRule -ResourceGroupName $rgname -NetworkManagerName $networkManagerName -ConfigName $RoutingConfigurationName -RuleCollectionName $RuleCollectionName -Name $RuleName -ForceDelete -PassThru -Force -AsJob;
```

Comment:
can you perform a Get after each delete to ensure they were deleted?

Reply:
Updated the PR with Get after each delete